python/rmm/_lib/device_buffer.pyx

```diff
@@ -11,27 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
 import numpy as np
 
-from libcpp.memory cimport unique_ptr
+from cpython.bytes cimport PyBytes_AS_STRING, PyBytes_FromStringAndSize
 from libc.stdint cimport uintptr_t
-
-from cpython.bytes cimport PyBytes_FromStringAndSize, PyBytes_AS_STRING
+from libcpp.memory cimport unique_ptr
 
 from rmm._lib.lib cimport (
     cudaError_t,
     cudaMemcpyDeviceToDevice,
     cudaMemcpyDeviceToHost,
     cudaMemcpyHostToDevice,
-    cudaSuccess,
     cudaStream_t,
-    cudaStreamSynchronize
+    cudaStreamSynchronize,
+    cudaSuccess,
 )
 
-cimport cython
-
+cimport cython  # isort:skip
```

Comment:
I think instead of doing the `# isort:skip` here we're better off just ignoring `E402` in the flake8 config. It logically makes sense to move this to the top.

Reply:
Here's the file: https://github.com/rapidsai/rmm/blob/branch-0.15/python/.flake8.cython

Reply:
Done.